docs/develop/api-reference/dictionary-collections.md

```diff
@@ -208,4 +208,25 @@ Sets several field:value pairs in a dictionary item. If the dictionary item does
 
 See [response objects](./response-objects.md) for specific information.
 
-</details>
\ No newline at end of file
+</details>
+
+### DictionaryLength
+Get the length of an existing dictionary item
```

Comment:
This is a bit ambiguous to me. I'd reword to "Get the number of entries in a dictionary item"